packages/protocol/contracts/common/MetaTransactionWallet.sol

```diff
@@ -198,24 +198,27 @@ contract MetaTransactionWallet is
    * @notice Executes multiple transactions on behalf of the signer.`
    * @param destinations The address to which each transaction is to be sent.
    * @param values The CELO value to be sent with each transaction.
-   * @param data The concatenated data to be sent in each transaction.
-   * @param dataLengths The length of each transaction's data.
+   * @param data The concatenated transaction data with their respective lengths <length+data>.
```

Comment:
Include exact specification of how this data is laid out (in particular, how many bytes the lengths take up).